pydocxs3upload/util/uri.py

```diff
@@ -32,7 +32,7 @@ def sanitize_filename(filename):
     by a dash (-) to make the image unique for round-tripping. In an effort to
     prevent a bunch of timestamps preceding the image name (in the event a
     document is round-tripped several times), strip off the timestamp
-    and dash. When images come from docx they are always `image\d+`. We only
+    and dash. When images come from docx they are always `image{N}`. We only
```

Comment:
pep8 was complaining about this. I don't know how to skip the documentation text so, I just changed it so it wouldn't trigger the pep8 scanner.